src/styles/index.scss

```
@@ -1,3 +1,15 @@
+@import "./blocks/variables";
+@import "./blocks/card.scss";
+@import "./blocks/stars.scss";
```

Please be consistent, either add extensions everywhere or remove them in this case
```suggestion
@import "./blocks/variables";
@import "./blocks/card";
@import "./blocks/stars";
```